packages/fxa-settings/src/components/VerifiedSessionGuard/index.tsx

```diff
@@ -12,7 +13,7 @@ export const VerifiedSessionGuard = ({
   children,
 }: {
   onDismiss: () => void;
-  onError: (error: Error) => void;
+  onError: (error: Error | ApolloError) => void;
```

Comment:
why not `onError: (error: Error | ApolloError | AuthenticationError)` ?

Reply:
Where is `AuthenticationError` thrown?

Reply:
Ohh, `AuthenticationError` is an error typed in Apollo Server, not Apollo Client. This is a nice article: https://www.apollographql.com/blog/full-stack-error-handling-with-graphql-apollo-5c12da407210/

Reply:
Ewww, are all GQL errors prepended with `GraphQL error: `? If so, since this is presented to the user, I am tempted to chop that beginning off.

Reply:
I am not sure. FWIW, this PR sorta relates to what Jared is doing in #6504